tools/ppcloader/include/gecko.h

```diff
@@ -41,14 +42,13 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #define gdprintf(...)
 #endif
 
-void CheckForGecko( void );
-void gprintf( const char *str, ... );
-void SetDumpDebug( u8 value );
-void InitGDBDebug( void );
+	void CheckForGecko(void);
```

Comment:
where are these tabs coming from?

Reply:
From the `extern "C" {` block :-) I've now fixed it.